test.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
+const path = require('path')
+const walk = require('walk-sync').entries
+const {nodeVersions} = require('./package.json')
+const semver = require('semver')
+const contentDir = path.join(__dirname, 'content')
+
+test('defines nodeVersions, a non-empty array of semver-valid versions', () => {
+  expect(Array.isArray(nodeVersions)).toBe(true)
+  expect(nodeVersions.length).toBeGreaterThan(1)
+  expect(nodeVersions.every(semver.valid)).toBe(true)
+  expect(nodeVersions.every(version => version.startsWith('v'))).toBe(true)
+})
+
+test('includes source English content for all nodeVersions', () => {
+  const fetchedVersions = fs.readdirSync(contentDir)
+  expect(nodeVersions).toEqual(fetchedVersions)
+})
```

Comment:
The test title refers to English content, I suggest checking the inner language folders
```
fetchedVersions.forEach(version => {
    const languages = fs.readdirSync(path.join(contentDir, version))
    expect(languages).include('en-US')
})